packages/tc-schema-validator/tests/type-test-suite.js

```diff
@@ -420,50 +422,54 @@ const relationshipTestSuite = type => {
 			properties,
 		});
 
-		describe('from', () => {
-			it('has existing from type', () => {
+		describe('relationship endpoints', () => {
+			it('uses existing record type fro from', () => {
 				const fromType = types.find(t => t.name === from.type);
 				expect(fromType).toBeDefined();
 				// assert the relationship node doesn't have more relationship
 				expect(isRichRelationshipType(fromType)).toBe(false);
 			});
-			it('relationship must be closed', () => {
-				const endType = types.find(t => t.name === to.type);
-				const propExists = Object.values(endType.properties).filter(
-					prop => prop.type === type.name,
-				);
-				expect(propExists.length).toBeGreaterThan(0);
-				// If from and to type are the same, we should validate direction
-				if (from.type === to.type) {
-					const closeRel = propExists.find(
-						prop => prop.direction === 'to',
-					);
-					expect(closeRel).toBeDefined();
-				}
-			});
-		});
 
-		describe('to', () => {
-			it('has existing to type', () => {
+			it('uses existing record type for to', () => {
```

Comment:
I found `'uses existing record type for to'` a bit vague ... are we saying type that already exists? how about `already defined`? 🤷‍♂